README.md

```diff
@@ -1,73 +1,67 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Capstone-Project - Online Conference Website
 
-# Project Name
-
-> Description the project.
+# Description the project.
 
+This project is example of Website for offering services to our clients
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+- html/css/js.
+- following linters standards.
+- following github flows standards.
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
+**My project works on mobile and desktop using @Media queries .**
+****************************************************
 To get a local copy up and running follow these simple example steps.
+follow>>>>>>>>>
 
-### Prerequisites
 
 ### Setup
 
-### Install
+to use the project please follow :
 
-### Usage
+ >Open GitHub and go to my GitHub repository name: Capstone-Project-I .to clone.
 
-### Run tests
+ >Click “Code” and copy the given URL.
 
-### Deployment
+ >Open “Git Bash” and change the current working directory to the location where you want the cloned directory.
 
+ >Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
 
+ >Syntax:
 
-## Authors
+  $ git clone {repository URL}
 
-👤 **Author1**
+### Usage
+this website can be used for commercial reasons or marketing.
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+### LIVE DEMO
+CHECK THIS [LINK](https://ogagaoghene.github.io/online-conference-website/)
 
-👤 **Author2**
+## Authors
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Esi Ogagaogene**
 
-## 🤝 Contributing
+- GitHub: [@Said-laasri](https://github.com/ogagaoghene)
+- Twitter: [@saidlaasr](https://twitter.com/@esi_ogagaoghene)
 
-Contributions, issues, and feature requests are welcome!
+## 🤝 credit
 
-Feel free to check the [issues page](../../issues/).
+[@Cindy Shin](https://www.behance.net/adagio07) Thanks for this amazing Design.
```

Comment:
- [ ] Kindly give the credits and appreciation under the `Acknowledgment` section below b because it's a project requirement.

![image](https://user-images.githubusercontent.com/68796232/175378994-27080e61-8294-454d-b8b9-bfd890464527.png)

Reply:
I have placed this at the acknowledgement section as requested